Assets/HoloToolkit/Stage/Scripts/StageManager.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.XR.WSA;
-using UnityEngine.Experimental.XR;
```

Comment:
This is required for the Boundary call on line 99. I haven't done a pass to wrap things in #if UNITY_UWP yet, so this isn't showing up properly.